xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_media.h

```diff
@@ -0,0 +1,188 @@
+    /// @brief __0000 0000__ : Nothing special to say about this mediaTag.
+    PVR_MEDIA_TAG_FLAG_UNDEFINED = 0,
+
+    /// @brief __0000 0001__ : This mediaTag is part of a series.
+    PVR_MEDIA_TAG_FLAG_IS_SERIES = (1 << 0),
+
+    /// @brief __0000 0010__ : This mediaTag will be flagged as new.
+    PVR_MEDIA_TAG_FLAG_IS_NEW = (1 << 1),
+
+    /// @brief __0000 0100__ : This mediaTag will be flagged as a premiere.
+    PVR_MEDIA_TAG_FLAG_IS_PREMIERE = (1 << 2),
+
+    /// @brief __0000 1000__ : This mediaTag will be flagged as a finale.
+    PVR_MEDIA_TAG_FLAG_IS_FINALE = (1 << 3),
+
+    /// @brief __0001 0000__ : This mediaTag will be flagged as live.
+    PVR_MEDIA_TAG_FLAG_IS_LIVE = (1 << 4),
+  } PVR_MEDIA_TAG_FLAG;
+///@}
+//----------------------------------------------------------------------------
+
+//============================================================================
+/// @ingroup cpp_kodi_addon_pvr_Defs_MediaTag_PVRMediaTag
+/// @brief Special @ref kodi::addon::PVRMediaTag::SetSeriesNumber() and
+/// @ref kodi::addon::PVRMediaTag::SetEpisodeNumber() value to indicate it is
+/// not to be used.
+///
+/// Used if mediaTag has no valid season and/or episode info.
+///
+#define PVR_MEDIA_TAG_INVALID_SERIES_EPISODE -1
+//----------------------------------------------------------------------------
+
+//============================================================================
+/// @ingroup cpp_kodi_addon_pvr_Defs_MediaTag_PVRMediaTag
+/// @brief Value where set in background to inform that related part not used.
+///
+/// Normally this related parts need not to set by this as it is default.
+#define PVR_MEDIA_TAG_VALUE_NOT_AVAILABLE -1
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  /// @defgroup cpp_kodi_addon_pvr_Defs_MediaTag_PVR_MEDIA_TAG_SECTION_TYPE enum PVR_MEDIA_TAG_SECTION_TYPE
+  /// @ingroup cpp_kodi_addon_pvr_Defs_MediaTag
+  /// @brief **PVR media tag section types**\n
+  /// Used on @ref kodi::addon::PVRMediaTag::SetSectionType() value to set related
+  /// type.
+  ///
+  ///@{
+  typedef enum PVR_MEDIA_TAG_SECTION_TYPE
+  {
+    /// @brief __0__ : Unknown type.
+    PVR_MEDIA_TAG_SECTION_TYPE_UNKNOWN = 0,
+
+    /// @brief __1__ : TV section.
+    PVR_MEDIA_TAG_SECTION_TYPE_TV = 1,
+
+    /// @brief __2__ : Radio section.
+    PVR_MEDIA_TAG_SECTION_TYPE_RADIO = 2,
+  } PVR_MEDIA_TAG_SECTION_TYPE;
+  ///@}
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  /// @defgroup cpp_kodi_addon_pvr_Defs_MediaTag_PVR_MEDIA_TAG_TYPE enum PVR_MEDIA_TAG_TYPE
+  /// @ingroup cpp_kodi_addon_pvr_Defs_MediaTag
+  /// @brief **PVR mediaTag types**\n
+  /// Used on @ref kodi::addon::PVRMediaTag::SetMediaType() value to set related
+  /// type.
+  ///
+  ///@{
+  typedef enum PVR_MEDIA_TAG_TYPE
```

Comment:
Would it make sense to add a "News" PVR_MEDIA_TAG_TYPE? Some news channels provide short news videos as VoD - not sure if it makes sense to tag them as movie or tv show.